analyses/01_ebp-assembly-workflow/nextflow.config

```diff
@@ -1,6 +1,32 @@
+// Include NBIS profiles
+includeConfig '../../envs/nextflow/nbis_profiles.config'
+profiles {
+    // Override nac profile for EBP processes
+    nac {
+        includeConfig '../../envs/nextflow/nac.config'
+        params {
+            // NAC does not have enough memory to have the db in ram. (Slows FCSGX down a lot.)
+            fcs.ramdisk_path = ''
+        }
+        process {
+            // Increase resourceLimits to use highest memory nodes on NAC
+            withName: 'FCSGX_RUNGX' {
+                cpus       = 16
+                memory     = 370.GB
+                time       = 30.d
+                resourceLimits = [
+                    cpus: 16,
+                    memory: 370.GB,
+                    time: 30.d
+                ]
+            }
+        }
+    }
+}
+
 // Add custom module configuration here
 
-// E.g Use high memory node for HIFIASM
+// E.g Use Rackham high memory node for HIFIASM
 // process {
 //     withName: 'HIFIASM' {
 //         cpus   = 20
```

Comment:
I guess this is also overriding the basic parameter for rackham for 'HIFIASM'

Reply:
What do you mean by basic parameter on Rackham? I put Rackham here because the `clusterOptions` extra spec is specific to Rackham.

Reply:
Yes that's what I meant that clusterOptions is specific to Rackham so all good